language/move-lang/tests/move_check/expansion/function_invalid_names.exp

```diff
@@ -1,34 +1,34 @@
-error[E02009]: invalid name
+error[E02010]: invalid name
   ┌─ tests/move_check/expansion/function_invalid_names.move:3:9
   │
 3 │     fun _foo() {}
   │         ^^^^ Invalid function name '_foo'. Function names cannot start with '_'
 
-error[E02009]: invalid name
+error[E02010]: invalid name
   ┌─ tests/move_check/expansion/function_invalid_names.move:4:9
   │
 4 │     fun _() {}
   │         ^ Invalid function name '_'. Function names cannot start with '_'
 
-error[E02009]: invalid name
+error[E02010]: invalid name
   ┌─ tests/move_check/expansion/function_invalid_names.move:5:9
   │
 5 │     fun ___() {}
   │         ^^^ Invalid function name '___'. Function names cannot start with '_'
 
-error[E02009]: invalid name
+error[E02010]: invalid name
    ┌─ tests/move_check/expansion/function_invalid_names.move:10:9
    │
 10 │     fun _foo() {}
    │         ^^^^ Invalid function name '_foo'. Function names cannot start with '_'
 
-error[E02009]: invalid name
+error[E02010]: invalid name
    ┌─ tests/move_check/expansion/function_invalid_names.move:13:9
    │
 13 │     fun _() {}
    │         ^ Invalid function name '_'. Function names cannot start with '_'
 
-error[E02009]: invalid name
+error[E02010]: invalid name
```

Comment:
Can you change the order of declaration of errors in the macro so that the error code doesn't change here?

This brings up a bigger question around what type of stability guarantees we want around these error codes. Should they be stable forever, or within major versions? My vote is for the latter, but interested on your thoughts on this.

Reply:
Changing the order here is only happening because it is the initial migration and I am trying to keep things well grouped. 

In the future, you will just append errors and in a way you get a neat little history of the language's development as the codes increase